pkg/config/config.go

```diff
@@ -139,3 +144,81 @@ func GetOLCollPullTimeout() time.Duration {
 	}
 	return timeout
 }
+
+// GetPeerRootPath returns the peer file system root path
+func GetPeerRootPath() string {
```

Comment:
I assume there is a TODO to refactor how config is handled/injected?

Reply:
Yeah, I'll add a TODO and create an issue.

Reply:
Cool - can you please link the issue?

Reply:
Here is the issue - #137